src/components/tooltip/Tooltip.css

```diff
@@ -89,3 +89,7 @@
   color: var(--grey-50);
   white-space: nowrap;
 }
+
+.tooltip .tooltipMarker {
```

Comment:
What's the reasoning for adding this line? In the past we have allowed tooltips to take up all the available width to show all of the information.